src/affix/index.jsx

```diff
@@ -62,6 +62,11 @@ class Affix extends React.Component {
         });
     }
 
+    componentWillReceiveProps(props) {
+        this.affixMode = this._getAffixMode(props);
+        this._updateNodePosition();
+    }
```

Comment:
This can cause potential performance problems, maybe you should add an `if...else...` statement to prevent unnecessary update, see other components for reference

Reply:
ok